example/ssd/dataset/iterator.py

```diff
@@ -123,17 +122,15 @@ def _get_batch(self):
                 index = self._index[self._current + i]
             # index = self.debug_index
             im_path = self._imdb.image_path_from_index(index)
-            img = cv2.imread(im_path)
+            with open(im_path, 'rb') as fp:
+                img_content = fp.read()
+            img = mx.img.imdecode(img_content)
```

Comment:
understand. the original code is operating against RBG. it converts image after cv2.imread via:
im[:, :, (0, 1, 2)] = im[:, :, (2, 1, 0)]